eZ/Publish/Core/FieldType/Image/ImageThumbnailStrategy.php

```diff
@@ -26,14 +30,19 @@ class ImageThumbnailStrategy implements FieldTypeBasedThumbnailStrategy
     /** @var string */
     private $variationName;
 
+    /** @var \Psr\Log\LoggerInterface */
+    private $logger;
+
     public function __construct(
         string $fieldTypeIdentifier,
         VariationHandler $variationHandler,
-        string $variationName
+        string $variationName,
+        LoggerInterface $logger = null
     ) {
         $this->fieldTypeIdentifier = $fieldTypeIdentifier;
         $this->variationHandler = $variationHandler;
         $this->variationName = $variationName;
+        $this->logger = null !== $logger ? $logger : new NullLogger();
```

Comment:
Without the extra configuration this logger is always gonna be a null logger.
Use `LoggerAwareTrait` and implement `LoggerAwareInterface` instead. Those combined together are [automatically configured](https://symfony.com/doc/5.4/logging.html#using-a-logger-inside-a-service) to inject an actual logger